examples/language-analyzer.go

```diff
@@ -26,14 +26,33 @@ var version = "alpha"
 func (*analyzer) NotifyReviewEvent(ctx context.Context, review *pb.ReviewEvent) (*pb.EventResponse, error) {
 	log.Infof("got review request %v", review)
 
-	conn, err := pb.DialContext(ctx, dataSrvAddr)
+	eh := func(err error, msg string) {
+		log.Errorf(err, msg)
+	}
+	conn, err := pb.DialContext(
+		ctx,
+		dataSrvAddr,
+		grpc.WithStreamInterceptor(grpchelper.CtxlogStreamClientInterceptor(eh)),
```

Comment:
let's add them by default similar to as we add `MsgSize` and `Insecure` by default

Reply:
oh. it actually expects function for error... let me think about it

Reply:
Ok, it makes sense. I'm gonna add these as defaults then 👍